app/content/react/library-reference/widgets.md

```diff
@@ -224,13 +284,12 @@ You can control the legend options through the following properties that must be
 
   const { myLayer } = useSelector((state) => state.carto.layers);
   const source = useSelector((state) => selectSourceById(state, myLayer?.source));
-  const cartoLayerProps = useCartoLayerProps({ source });
+  const cartoLayerProps = useCartoLayerProps({ source, layerConfig });
 
   if (myLayer && source) {
     return new CartoLayer({
       ...cartoLayerProps,
       id: MY_LAYER_ID,
```

Comment:
FYI, `id` is also provided from cartoLayerProps.

Reply:
Removed.